packages/web-components/src/components/card-in-card/card-in-card.ts

```diff
@@ -48,7 +48,7 @@ class DDSCardInCard extends StableSelectorMixin(DDSCardCTA) {
     if (linkNode) {
       linkNode.classList.add(`${prefix}--card-in-card`);
     }
-    this.onclick = () =>
+    this.querySelector('dds-card-in-card-image').onclick = () =>
```

Comment:
```suggestion
    this.querySelector('dds-card-in-card-image')?.onclick = () =>
```

Reply:
@ariellalgilmore Not sure if this ternary operator suggestion works here, but I went ahead and added `if` conditional check. let me know what you think.

Reply:
i think we can stick with the ternary but looks like ci-checks don't like the `onclick`, so let's try this

```
this.querySelector(`${ddsPrefix}-card-in-card-image`)?.addEventListener('onclick', () => {
      this.querySelector(`${ddsPrefix}-card-cta-footer`)
        ?.shadowRoot?.querySelector(`a`)
        ?.click();
    });
```